pkg/log/format.go

```diff
@@ -71,6 +73,9 @@ func (l *logger) getWriter(format string) OktetoWriter {
 		l.outputMode = JSONFormat
 		l.out.SetFormatter(&JSONLogFormat{})
 		return newJSONWriter(l.out, l.file)
+	case SilentFormat:
```

Comment:
Perhaps it's more explicit if we call it "Log File" or something related? 

```suggestion
	case LogFileFormat:
```

Reply:
🤔 giving a thought to this, for me it makes sense "silent" as is not displaying output.... but, is saving it to a file. We are using this as option internally, so, i would keep the name but perhaps add couple more description to the Format (line 61) or here in the case (line 76) to explain why and how the logs are kept

Reply:
I prefer to use the `Silent` naming but definitely this is something we need to join and discuss about it (everything related to logs).